app/services/feature_service.rb

```diff
@@ -13,7 +13,7 @@ def enabled?(feature_name)
 
       segments = feature_name.to_s.split('.')
 
-      segments.reduce(Settings.features) { |config, segment| config[segment] }
+      segments.reduce(Settings.features) { |config, segment| config[segment] } == true
```

Comment:
Why this change?

Reply:
The method signature is

`    def enabled?(feature_name)`

So it shoud only return a true or a false given enabled`?` 

Without it any value can be returned including `nil`